test/eslintCompat.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+const eslint = require('eslint');
+
+// Small compat layer to support ESLint v8 API on both v7 and v8
+class ESLintCompat {
+    constructor(options) {
+        Object.assign(options, options.overrideConfig);
+        delete options.overrideConfig;
+        this.eslint = new eslint.CLIEngine(options);
```

Comment:
I don't understand why this is needed. The `eslint.CLIEngine` was already [deprecated](https://eslint.org/docs/7.0.0/developer-guide/nodejs-api#cliengine) in eslint@7. 
Both 7 and 8 share the same `eslint.ESLint` API.

Reply:
Ah, I didn't realize that. OK I'll get rid of the compat API; that was silly of me not to check 😂 .